src/components/Participant/RegistrationProcess/StartRegistrationProcess/StartRegistrationProcess.tsx

```diff
@@ -40,6 +40,9 @@ function StartRegistrationProcess() {
         </DTEContent>
         <br></br>
         <br></br>
+        <DTEButton $outlined onClick={() => history.push("/Participants/introduction")}>
+          DEV BUTTON
```

Comment:
whats this for?

Reply:
Oops forgot to remove that, I put it there to navigate to that page because I never knew how to another way